report.go

```diff
@@ -43,9 +43,17 @@ func report(results []*RunResult) {
 		if i == 0 {
 			reportFile.Title = folderPath
 		}
+
 		reportFile.Data[i].Name = name
 		reportFile.Data[i].HDR = read(filepath.Join(folderPath, name+".hdr"))
-		reportFile.Data[i].JSON = mustJSONUnmarshal(filepath.Join(folderPath, name+".json"))
+		json := mustJSONUnmarshal(filepath.Join(folderPath, name+".json"))
+
+		relayMetrics := json.RelayMetrics
+		if len(relayMetrics) != 0 {
+			reportFile.RelayMetrics = relayMetrics
+		}
```

Comment:
This is sort of tricky that it is in a loop but relies on the baseline/instrumented differences.
Wanted to share that it is more idiomatic to check for nil when we don't really care about the length.